designer/server/src/models/create.js

```diff
@@ -38,45 +55,7 @@ export function organisationViewModel(metadata, validation) {
       legend: {
         text: 'Choose a lead organisation for this form'
       },
-      items: [
-        {
-          text: 'Animal and Plant Health Agency – APHA',
-          value: 'Animal and Plant Health Agency – APHA'
-        },
-        {
-          text: 'Centre for Environment, Fisheries and Aquaculture Science – Cefas',
-          value:
-            'Centre for Environment, Fisheries and Aquaculture Science – Cefas'
-        },
-        {
-          text: 'Defra',
-          value: 'Defra'
-        },
-        {
-          text: 'Environment Agency',
-          value: 'Environment Agency'
-        },
-        {
-          text: 'Forestry Commission',
-          value: 'Forestry Commission'
-        },
-        {
-          text: 'Marine Management Organisation – MMO',
-          value: 'Marine Management Organisation – MMO'
-        },
-        {
-          text: 'Natural England',
-          value: 'Natural England'
-        },
-        {
-          text: 'Rural Payments Agency – RPA',
-          value: 'Rural Payments Agency – RPA'
-        },
-        {
-          text: 'Veterinary Medicines Directorate – VMD',
-          value: 'Veterinary Medicines Directorate – VMD'
-        }
-      ],
+      items: organisationItems,
```

Comment:
I've suggested a TypeScript fix, should be able to stick the mapping straight into here now

Reply:
I've inlined this to `items: organisations.map()` etc

Reply:
I pullled it outside so as not to incur the cost each time the form is loaded, but agree the code looks better for it inside. Probably not worth the micro-optimisation :)

Thanks for merging :+1:

Reply:
Ah assumed it was to apply the JSDoc (separately) to try and fix those type issues!